packages/components/Input/src/Input.tsx

```diff
@@ -48,60 +35,9 @@ export const Input = compose<InputType>({
     assistiveText: Text,
     accessoryText: Text,
   },
-  useRender: (userProps: InputProps, useSlots: UseSlots<InputType>) => {
-    const input = useInput(userProps);
-    const accessoryIconProps = createIconProps(input.props.accessoryIcon);
-    const Slots = useSlots(userProps, (layer) => inputLookup(layer, input.state, userProps));
-
-    return (final: InputProps) => {
-      const {
-        label,
-        accessoryText,
-        assistiveText,
-        defaultIcon,
-        accessoryIcon,
-        textInputProps,
-        error,
-        accessoryButtonOnPress,
-        keyboardShouldPersistTaps,
-        iconProps,
-        componentRef,
-        accessoryIconAccessibilityLabel,
-        onChange, // Remove out of mergedProps
-        ...mergedProps
-      } = mergeProps(input.props, final);
-      const IconWrapper = defaultIcon ? Slots.inputWrapper : Fragment;
-
-      return (
-        <Slots.root {...mergedProps} keyboardShouldPersistTaps={keyboardShouldPersistTaps ? keyboardShouldPersistTaps : 'handled'}>
-          {label && <Slots.label>{label}</Slots.label>}
-          <IconWrapper>
-            {defaultIcon && <Slots.icon {...iconProps} accessible={false} />}
-            <Slots.input>
-              <Slots.textInput {...textInputProps} />
-              {accessoryText && !!input.state.text && <Slots.accessoryText>{accessoryText}</Slots.accessoryText>}
-              {accessoryIcon && !!input.state.text && (
-                <Slots.accessoryIconPressable
-                  accessibilityRole="button"
-                  accessibilityLabel={accessoryIconAccessibilityLabel}
-                  onPress={(e) => {
-                    if (accessoryButtonOnPress) accessoryButtonOnPress(e);
-                    else {
-                      if (input.props.textInputProps.editable === false) return;
-                      input.props.setText('');
-                      onChange && onChange('');
-                      componentRef.current.focus();
-                    }
-                  }}
-                >
-                  <Slots.accessoryIcon {...accessoryIconProps} accessible={false} />
-                </Slots.accessoryIconPressable>
-              )}
-            </Slots.input>
-          </IconWrapper>
-          {(assistiveText || error != undefined) && <Slots.assistiveText>{error ? error : assistiveText}</Slots.assistiveText>}
-        </Slots.root>
-      );
+  useRender: (_userProps: InputProps, _useSlots: UseSlots<InputType>) => {
+    return (_final: InputProps) => {
+      return null;
```

Comment:
?

Reply:
Disabling for other platforms similar to Chip